extensions/wikia/ThemeDesigner/UploadWordmarkFromFile.class.php

```diff
@@ -1,59 +1,60 @@
 <?php
+
 /*
  * @author Federico "Lox" Lucignano
  */
 
 class UploadWordmarkFromFile extends UploadFromFile {
 	const FILEDIMENSIONS_ERROR = 30;
 	const FILETYPE_ERROR = 31;
-	
-	public function verifyUpload(){
+
+	public function verifyUpload() {
 		$this->getTitle(); //will fill in final destination and extension
 		$details = parent::verifyUpload();
-		
-		if ( $details[ 'status' ] == self::OK ){ 
+
+		if ( $details['status'] == self::OK ) {
 			// check file type (just by extension)
-			
-			if ( $this->checkFileExtension( $this->mFinalExtension, array( 'png' ) ) ) {
+
+			if ( $this->checkFileExtension( $this->mFinalExtension, [ 'png' ] ) ) {
 				$imageSize = getimagesize( $this->getTempPath() );
 
-				if($imageSize[0] > 250 || $imageSize[1] > 65) {
-					$details[ 'status' ] = self::FILEDIMENSIONS_ERROR;
+				if ( $imageSize[0] > 250 || $imageSize[1] > 65 ) {
+					$details['status'] = self::FILEDIMENSIONS_ERROR;
 				}
 			} else {
-				$details[ 'status' ] = self::FILETYPE_ERROR;
+				$details['status'] = self::FILETYPE_ERROR;
 			}
 		}
 
 		return $details;
 	}
-	
-	
+
+
 	public function performUpload() {
 		global $wgUser;
-		
-		return parent::performUpload('', '', false, $wgUser);
+
+		return parent::performUpload( '', '', false, $wgUser );
 	}
-	
+
 	public function getLocalFile() {
-		if( is_null( $this->mLocalFile ) ) {
+		if ( is_null( $this->mLocalFile ) ) {
 			//TODO: find out what namespace constant 6 is
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)